cql/ManageRareAbnormality.cql

```diff
@@ -1992,7 +1992,7 @@ define RecommendationForSurveillanceAfterAbnormalities:
       InitialIntensiveSurveillancePeriod
     ) then
       {
-        short: 'See Details',
+        short: 'Surveillance 3-year follow-up',
         date: Today(),
         group: 'Management Surveillance (J.3.2)',
         details: {
```

Comment:
Should details say "Surveillance with primary HPV testing or cotesting" to match other recommendations (e.g. I.3)